packages/pigeon/platform_tests/ios_unit_tests/ios/Runner/async_handlers.h

```diff
@@ -0,0 +1,27 @@
+// Autogenerated from Pigeon (v0.1.19), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+#import <Foundation/Foundation.h>
+@protocol FlutterBinaryMessenger;
+@class FlutterError;
+@class FlutterStandardTypedData;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class Value;
```

Comment:
Can this generated class be prefixed, too?

Reply:
It can.  It's an argument to the invocation of pigeon.  For this test I haven't specified a prefix.  There are other tests covering the prefix.